proposals/lip-0062.md

```diff
@@ -78,3 +78,30 @@ TBD
 This LIP results in a hard fork as nodes following the proposed protocol will reject signatures according to the previous protocol, and nodes following the previous protocol will reject signatures according to the proposed protocol.
 
 [lip-0008]: https://github.com/LiskHQ/lips/blob/main/proposals/lip-0008.md
+
+## Appendix
+
+### Test Vectors
+
+The following tests supersede the corresponding tests given in [LIP 0038](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0038.md#test-vectors).
+
+#### `signBLS`
+
+```python
+sk = 0x263dbd792f5b1be47ed85f8938c0f29586af0d3ac7b977f21c278fe1462040e3
+tag = b"LSK_TX_"
+chainID = 0x00000000
+message = 0xbeaf
+signBLS(sk, tag, chainID, message) = 0x80c3da661b5bb80bb841367255f7b087b969c075661895b7ac8b74b72360be54693b3485eff7d816924517a21ef1c3a30a8f9402572d5a63a7ff2f71ca6929a8c3d7f75fd72edd1aa478ecc09966a133e829600f0111a1e40bbe35db61e8c689
+```
+
+#### `verifyBLS`
+
+```python
+pk = 0xa491d1b0ecd9bb917989f0e74f0dea0422eac4a873e5e2644f368dffb9a6e20fd6e10c1b77654d067c0618f6e5a7f79a
+tag = b"LSK_TX_"
+chainID = 0x00000000
+message = 0xbeaf
+sig = 0x80c3da661b5bb80bb841367255f7b087b969c075661895b7ac8b74b72360be54693b3485eff7d816924517a21ef1c3a30a8f9402572d5a63a7ff2f71ca6929a8c3d7f75fd72edd1aa478ecc09966a133e829600f0111a1e40bbe35db61e8c689
+verifyBLS(pk, tag, chainID, message, sig) = VALID
```

Comment:
- Don't we assume now that the function returns a Boolean?
- Shouldn't it be `==` instead of `=`?